### ISSUE TITLE
Zuul VMs are deployed with SSH Keys from SCS Community Members

### ISSUE THREAD

**Issue by o-otte** [SovereignCloudStack/issues#433]
The SCS Zuul VMs needs to be accessible by multiple people to ensure maintenance can be done by more than one.

## Definition of Ready:

- [ ] User Story is small enough to be finished within one sprint
- [ ] User Story is clear and understood by the whole team
- [ ] Acceptance criteria are defined
- [ ] Acceptance criteria are clear and understood by the whole team

## Definition of Done:

- [ ] SSH Keys of a subset of SCS Community Members are deployed in an automated fashion
- [ ] All acceptance criteria are met
- [ ] Changes have been reviewed
- [ ] CI tests have run successfully
- [ ] Documentation has been updated
- [ ] Release Notes have been updated

**Comment by berendt** [comment]
Good point. I have that on the todo list for our osism.commons.operator role. I'll add it there shortly. Don't know if you use it, but if you do you can use it directly.
